### ISSUE TITLE
MPLasso executable script

### ISSUE THREAD

**Issue by VadimDu** [ChiehLo/MPLasso_RPackage#1]
Dear developers,
I wanted to ask if you plan to release soon an executable-script version of MPLasso?
I would be very happy to try your method on our data, but you do not provide any usage instructions as to how run your codes? 
Even the Demo_HMP.R does not have any comments as to explain the different steps you perform, making it hard to follow and thus used by other researchers, even ones that understand programming.

If you want to make your method available for many other researchers to use (including myself), I would recommend adding a detailed running instructions or converting the code to command-line executable.

Thank you very much
Best regards
Vadim Dubinsky

**Comment by ChiehLo** [comment]
Hi, 
Thank for your interest. I will update it shortly and let you know. 

Best, 
Chieh

**Comment by VadimDu** [comment]
Hello,

Any updates?

Thank you

**Comment by zoey-rw** [comment]
Hi @ChiehLo, thank you for sharing your code online. I am very interested in implementing your algorithm, but I am not sure how to apply it to my dataset. Do you have any plans to add instructions to this repo?

Thank you again!
Zoey
